src/styles/Popup_modal.scss

```diff
@@ -0,0 +1,191 @@
+/* 
+#FFFFFF, Text
+#FFFFFF, 70% Opacity Secondary Text
+#FFFFFF, 20% Opacity Lines
+border-radius: 10px;
+#211E23 Background
+#3D3D4C Window Background
+#48485A Input Background
+#50A0CD Button
+*/
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap');
+
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .modal-content {
+    display: flex;
+    flex-direction: column;
+    overflow-y: scroll;
+    width: 450px;
+    height: 50%;
+    background-color: #3d3d4c;
+    padding: 20px;
+    border-radius: 10px;
+
+    button.modal-close-button {
+      position: fixed;
+      align-self: flex-end;
+      background-color: #ff000080;
+      color: #ffffff;
+      border-radius: 32px;
+      padding: 3px 8px;
+      border: none;
+      font-weight: 700;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ff0000;
+      }
+    }
+    .user-post-name {
+      font-weight: bold;
+      text-decoration: none;
+      color: #ffffff;
+      font-size: 1.2rem;
+    }
+    .user-post-body {
+      color: #ffffff;
+      word-wrap: break-word;
+      font-weight: 300;
+    }
+    .comment-card {
+      padding-bottom: 1rem;
+      border-bottom: 1px solid #ffffffb3;
+      .comment-user-name {
+        margin-top: 1rem;
+        font-weight: bold;
+        text-decoration: none;
+        color: #ffffff;
+        font-size: 1rem;
+      }
+      .comment-body {
+        color: #ffffff;
+        word-wrap: break-word;
+        font-weight: 300;
+      }
+      .comment-date {
+        font-size: 0.8rem;
+        color: #ffffff4d;
+      }
+    }
+
+    form.new-comment {
+      position: fixed;
+      top: auto;
+      bottom: 18%;
+      display: flex;
+      margin-bottom: 1rem;
+      .new-comment-input {
```

Comment:
Maybe add in `border: none;`?

Reply:
removed borders on both edit posts and post comments